internal/bundle/utils.go

```diff
@@ -108,6 +110,22 @@ func LoadBundleFromDisk(path, name string, bvc *bundle.VerificationConfig) (*bun
 		if err != nil {
 			return nil, err
 		}
+
```

Comment:
We can read the manifest file first and get the `etag` value and pass that to the `bundle.NewCustomReader` call via `WithBundleEtag()`.

Reply:
Thanks Ash, I'll have a go at this now.

Reply:
Added in 3899e770b